.github/workflows/upload_coverage_report.yml

```diff
@@ -27,9 +27,18 @@ jobs:
           --workspace
           --lcov
           --output-path "${{ env.REPORT_NAME }}"
-      - name: Upload coverage to codecov
-        uses: codecov/codecov-action@v3
+      - name: Upload coverage to codecov (tokenless)
+        if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repositor
```

Comment:
Is this a typo? `github.repositor`

Reply:
Yes, this was a typo. And I have no idea why did it worked.